src/js/components/AppStatusComponent.jsx

```diff
@@ -29,6 +30,10 @@ var AppStatusComponent = React.createClass({
     showSummary: React.PropTypes.bool
   },
 
+  shouldComponentUpdate: function (nextProps) {
```

Comment:
Same thing here with model.tasks, can become huge.
You could compare directly on the model-propteries used here.